api/common/apiaddresser.go

```diff
@@ -39,21 +39,6 @@ func (a *APIAddresser) APIAddresses() ([]string, error) {
 	return result.Result, nil
 }
 
-// ModelUUID returns the model UUID to connect to the model
-// that the current connection is for.
-//
-// TODO(axw) this has bugger all to do with addresses, and
```

Comment:
Note the comment. This is why we got bitten.

Reply:
You should share this more directly with the team.  It's a good example of why not to do it.